### PR TITLE
- Renamed dlint to flake8 and removed the DUO options as these are no…

### PR DESCRIPTION
**Pull request by pieterge**
- Touched up the docstrings for the return statements as they were incorrect and inconsistent
- Touched up the gitignore file as requested in review
- changed reference of linenrs to line_nrs to improve readability. The original review mentioned level.py, but there was no mention of this variable, so I have replaced the mentions throughout the project.

### CODE REVIEW

**Comment by pieterge** [comment]
Ow, I re-ran Sphinx too, and the documentation seems to be cleaned up :)

File: .gitignore [hunk 1]
```diff
@@ -1,6 +1,3 @@
-
-README.rst
-
 # DepHell stuff
 poetry.lock
```
Comment (vfxGer) [inline .gitignore:2]: We commit the poetry.lock file to git, the reasons for this are outlined here https://python-poetry.org/docs/basic-usage/#commit-your-poetrylock-file-to-version-control

Comment (pieterge) [reply .gitignore]: Done

**Review by vfxGer — APPROVED** [review]
LGTM

**Pull request merged**